CMakeLists.txt

```diff
@@ -9,29 +9,25 @@ project(editorconfig-core-qt
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
-find_package(QT NAMES Qt6 Qt5 COMPONENTS Core REQUIRED)
-find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core REQUIRED)
-
-enable_testing()
-set(EDITORCONFIG_CMD ${CMAKE_BINARY_DIR}/src/editorconfig-app/editorconfig-app)
-add_subdirectory(tests)
-
-if (BUILD_STATIC)
-    set(CMAKE_STATIC_LIBRARY_SUFFIX "_static${CMAKE_STATIC_LIBRARY_SUFFIX}")
-    add_library(editorconfig-core-qt STATIC
-        EditorConfig.cpp
-        EditorConfig.h
-    )
-else()
-    add_library(editorconfig-core-qt SHARED
-        EditorConfig.cpp
-        EditorConfig.h
-    )
-endif()
+find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Core)
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Core)
+
+add_library(editorconfig-core-qt
```

Comment:
yap, not I cannot control as a consumer of this project - which kind of library I want. Do I want a DLL? or *.a?

Reply:
This requires additional code changes...plus I dont know of anyone using a DLL. So for now I'd like to focus just on getting cmake to replace qmake. It was adding some complexity that made troubleshooting some of this difficult. Plus it looks like [`BUILD_SHARED_LIBS`](https://cmake.org/cmake/help/latest/variable/BUILD_SHARED_LIBS.html) is the recommended way to specify what to build.